executor/aggregate.go

```diff
@@ -63,6 +66,7 @@ func newBaseHashAggWorker(ctx sessionctx.Context, finishCh <-chan struct{}, aggF
 type HashAggPartialWorker struct {
 	baseHashAggWorker
 
+	hashAgg           *HashAggExec
```

Comment:
hashAgg --> self?

Reply:
I think we can use `stats *HashAggRuntimeStat` instead~